### ISSUE TITLE
[BUG] Linear (OAuth) and Linear (API) - Create Issue throw error "this.client(...).issueCreate is not a function"

### ISSUE THREAD

**Issue by vunguyenhung** [PipedreamHQ/pipedream#5170]
**Describe the bug**
Failed to use Linear (OAuth) and Linear (API) - Create Issue action since it throws error "this.client(...).issueCreate is not a function"
<img width="727" alt="image" src="https://user-images.githubusercontent.com/18277920/212218403-6b460781-2730-4621-8fd7-a34a4c3e1e4f.png">

**To Reproduce**
1. Create a new workflow
2. Add a new Linear (OAuth) action, test it with sample data
<img width="727" alt="image" src="https://user-images.githubusercontent.com/18277920/212218403-6b460781-2730-4621-8fd7-a34a4c3e1e4f.png">
2. Add a new Linear (API) action, test it with sample data
<img width="729" alt="image" src="https://user-images.githubusercontent.com/18277920/212218842-d7988d32-6345-40db-966b-4b7542b320b1.png">

**Expected behavior**
The action should be executed successfully, the issue should be created on Linear.
Please also check other actions for similar issues

**Additional context**
First reported by the customer here: https://pipedream-users.slack.com/archives/CPTJYRY5A/p1673559040084019

**Comment by andrewjschuang** [comment]
TLDR - @dannyroosevelt the `@linear/sdk` package version in [package.json](https://github.com/PipedreamHQ/pipedream/blob/master/components/linear_app/package.json#L14) is not being pinned. The runtime is using the latest version available - is this a bug?

---

The customer reported that the step _**just started throwing issues out of the blue**_.

Took a quick look at this, and it seems the `@linear/sdk` published a new version - `2.0.0` - 16 hours ago (as of now). 

Although in our `package.json` file, we do pin the version to `1.22.0`, but for some reason the action runtime is using the latest version of the sdk.

These images below show different errors if we pin different versions:

<img width="789" alt="Screenshot 2023-01-13 at 8 29 12 AM" src="https://user-images.githubusercontent.com/15385076/212311121-44e1b16c-c2ca-43c4-b0d1-83c17071c9dc.png">

<img width="789" alt="Screenshot 2023-01-13 at 8 29 40 AM" src="https://user-images.githubusercontent.com/15385076/212311139-a2005469-b133-4c18-bd7c-2e58e300bbe6.png">

**Comment by js07** [comment]
If I’m not mistaken, components use the latest version of a package rather than the version pinned in the app’s package.json.
I think the Linear components can be updated to either work with the latest version of `@linear/sdk`, or pin the version in code: `import { LinearClient } from "@linear/sdk@1.22.0";`.
